### DISCUSSION TITLE
How to lower block unlock interval?

### DISCUSSION THREAD

**Discussion by MiningCryptoLive** [coinfoundry/miningcore#1212] [Q&A]
I am running an Ergo pool using Miningcore and it works great. I am trying to figure out how to lower the block unlock time from 720 (almost 24hrs) down to 70 so payouts to miners are quicker. Any help would be appreciated.

**Comment by oliverw** [comment]
```
"pools": [
  {
    "id": "ergo",
    "paymentprocessing": {
      "minimumConfirmations": 70 
    }
  }
]
```

**Reply by MiningCryptoLive** [reply]
Thank you. One other question. Is there a way to add round variance to the pool?

**Reply by oliverw** [reply]
Elaborate please.

**Reply by MiningCryptoLive** [reply]
I am trying to add current round variance to the home page
![round-variance](https://user-images.githubusercontent.com/1049926/159128373-922acd16-c92b-4003-a589-4671c9392b2a.png)
.

**Reply by oliverw** [reply]
Something like this?
https://miningcore.pro/pool/rtm-solo (switch to Blocks Tab)

**Reply by MiningCryptoLive** [reply]
Yes sir

**Reply by MiningCryptoLive** [reply]
under the pool stats - Block Effort

**Reply by oliverw** [reply]
The API exposes that info already: `/api/pools/<poolid>/blocks`. Property `effort`

**Reply by oliverw** [reply]
Take the average of the last N blocks and you got that value.

**Reply by MiningCryptoLive** [reply]
I'm new to all this. I found the api exposes it, just trying to figure out how to add it to the idex.html.

**Reply by MiningCryptoLive** [reply]
I have one other question if you can help it would be much appreciated. I want to add an EU server to the pool. Can you please give me an idea on how to do that? If not it's fine. Thanks for everything
